support-frontend/assets/pages/new-contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -173,15 +171,9 @@ function goalReachedTemplate() {
   return null;
 }
 
-function urlSpecificDetails() {
-  if (getQueryParameter('ticker') === 'true') {
-    return {
-      tickerJsonUrl: '/ticker.json',
-    };
-  }
-
-  return {};
-}
+const showTicker: boolean = getQueryParameter('ticker') === 'true';
+const campaignName = isFrontlineCampaign() ? 'thefrontline' : '';
```

Comment:
use the `getCampaignName()` function here

Reply:
👍

Reply:
yep, and delete references to `thefrontline` from it